woof-code/rootfs-skeleton/usr/sbin/bootmanager

```diff
@@ -346,13 +391,38 @@ else
 	XML_ACTION='sfs_load &'
 	SFSFRAME_XML="$(generic_xml)" #--
 fi
+case $KARCH in
+	x86_64|i?86|amd64);;
```

Comment:
It's always i?386 or x86_64, never amd64 AFAIK

Reply:
Yeah we don't use $DEBARCH